codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeDecoder.java

```diff
@@ -204,7 +205,9 @@ private State readHeaders(ByteBuf buffer, StompHeaders headers) {
                 } else {
                     int globalIndex = indexOf(buffer, buffer.readerIndex(),
                             buffer.writerIndex(), StompConstants.NUL);
-                    if (globalIndex != -1) {
+                    if (globalIndex == -1) {
+                        throw Signal.valueOf(ReplayingDecoder.class, "REPLAY");
```

Comment:
it should just return null and then break out of the decode loop without adding anything to the out list.